src/fs-access/file-open.mjs

```diff
@@ -29,19 +29,19 @@ export default async (options = [{}]) => {
   if (!Array.isArray(options)) {
     options = [options];
```

Comment:
I'd prefer not to mutate parameters

Reply:
What do you propose as an alternative if the function should accept arrays and single values at the same time?

Reply:
`const opts = Array.isArray(options) ? options : [options];`

Reply:
Sure, but what would be the advantage? Honestly curious.